### DISCUSSION TITLE
Translation IT (not all)

### DISCUSSION THREAD

**Discussion by VeUni45** [aclap-dev/video-downloadhelper#895] [Language & Translation]
[it.txt](https://github.com/user-attachments/files/15887600/it.txt)

**Comment by Yu-Kanda** [comment]
IT complete translation file attached,
[it.txt](https://github.com/user-attachments/files/15906562/it.txt)

**Comment by EuroPDC** [comment]
Missing IT translations here: [it.txt](https://github.com/user-attachments/files/15943968/it.txt)

Differently from the other I added the translation of "v9_settings_checkbox_thumbnail_in_notification".

**Comment by VoloLibero** [comment]
[it.txt](https://github.com/user-attachments/files/15981053/it.txt)
Here the newest Italian language file
